src/Bounds.cpp

```diff
@@ -2061,17 +2061,21 @@ class BoxesTouched : public IRGraphVisitor {
                     const GT *gt = solved.as<GT>();
                     const GE *ge = solved.as<GE>();
                     const EQ *eq = solved.as<EQ>();
-                    Expr rhs;
-                    if (lt) {rhs = lt->b;}
-                    if (le) {rhs = le->b;}
-                    if (gt) {rhs = gt->b;}
-                    if (ge) {rhs = ge->b;}
-                    if (eq) {rhs = eq->b;}
+                    Expr lhs, rhs;
+                    if (lt) {lhs = lt->a; rhs = lt->b;}
```

Comment:
super-nit: would it be worthwhile to make these if-else since they are all mutually exclusive?